### PR TITLE
feat: quote section

### PR DESCRIPTION
**Pull request by muskaanshah**
features added: fetching quote from API,
quote refreshes only after 24 hours,
time format isnt 24 hour format when refreshed, it depends on the previous state set by user
name change of user

### CODE REVIEW

File: src/pages/landingpage.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .landingimage {
 	height: 100vh;
-	background: url("../assets/wallpaper.jpg");
+	/* background: url("../assets/wallpaper.jpg"); */
```
Comment (ScoobyDubeyDo) [inline src/pages/landingpage.css:3]: **must**: remove  unused code

**Review by ScoobyDubeyDo — APPROVED** [review]

File: src/pages/Landing.jsx [hunk 2]
```diff
@@ -35,9 +37,15 @@ function Landing() {
...
 		}
 	}
+	const navigate = useNavigate();
+	const changeNameHandler = () => {
+		localStorage.removeItem("name");
+		navigate("/");
+	};
 	useEffect(() => {
 		const user = localStorage.getItem("name");
 		dispatch({ type: "SET_USERNAME", payload: { value: user } });
```
Comment (shraddha-amber) [inline src/pages/Landing.jsx:51]: suggestion: might create an object in constants for storing all the actions and passing the variable here

File: src/pages/Landing.jsx [hunk 3]
```diff
@@ -35,9 +37,15 @@ function Landing() {
...
 			localStorage.removeItem("Focus");
 			localStorage.removeItem("Todos");
+			localStorage.removeItem("Quote");
 			localStorage.setItem("setupTime", day);
 		}
 	}
+	const navigate = useNavigate();
+	const changeNameHandler = () => {
+		localStorage.removeItem("name");
+		navigate("/");
```
Comment (shraddha-amber) [inline src/pages/Landing.jsx:47]: suggestion: could save all the routes in constants too

**Review by shraddha-amber — APPROVED** [review]
Clean and small PR, great work!

**Pull request merged**